mopro-ffi/tests/test_generated_bindings.rs

```diff
@@ -1,4 +1,6 @@
 uniffi::build_foreign_language_testcases!(
+    // "tests/bindings/test_mopro_gen_benchmarks_report.swift", // uncomment this to generate the benchmarks report in swift
+    "tests/bindings/test_mopro_gpu_benchmarks.swift",
```

Comment:
I think you can gate this whole block with the feature flag

Reply:
Thank you. I added a feature flag for this function. (seems like it is not allow to add a feature flag inside the function parameters)